homeassistant/components/smhi/weather.py

```diff
@@ -115,6 +119,7 @@ class SmhiWeather(WeatherEntity):
     _attr_temperature_unit = TEMP_CELSIUS
     _attr_visibility_unit = LENGTH_KILOMETERS
     _attr_precipitation_unit = LENGTH_MILLIMETERS
+    _attr_wind_speed_unit = SPEED_METERS_PER_SECOND
```

Comment:
Wind speed should be reported in km/h until we've decided otherwise in an architecture discussion.

https://developers.home-assistant.io/docs/core/entity/weather#properties

Reply:
I was searching for it yesterday but could not find such discussion.
Then I looked into the code and found that these units have been added along the way so should be good to go from my perspective.
Now I look [here](https://github.com/home-assistant/core/pull/59533) and see maybe that isn't the case so how do we go from here?
Open a PR and ADR discussion to finally resolve this topic as it's been going on for quite some time and nothing seems to happen to it?

Reply:
The first step is to open or have the ADR discussion. I can't promise that it'll resolve quickly though. That depends on interest from the members community.

Reply:
https://github.com/home-assistant/architecture/discussions/772

Reply:
I set this PR as draft for the time being until ADR discussion is then resolved